flink-connectors/flink-hbase/src/test/java/org/apache/flink/addons/hbase/HBaseTestingClusterAutostarter.java

```diff
@@ -211,8 +211,7 @@ private static void createHBaseSiteXml(File hbaseSiteXmlDirectory, String zookee
 
 	private static void addDirectoryToClassPath(File directory) {
 		try {
-			// Get the classloader actually used by HBaseConfiguration
-			ClassLoader classLoader = HBaseConfiguration.create().getClassLoader();
```

Comment:
The point of this method is to mutate the default classloader so that the hbase-site.xml is on the classpath later on.

By creating a new classloader that is immediately discarded once the method exists and never used to actually load anything this method becomes a no-op, which isn't acceptable.